labelme/app.py

```diff
@@ -567,6 +573,26 @@ def __init__(
         )
         fill_drawing.trigger()
 
+        sort_shapes_by_id = action(
```

Comment:
As we discussed you can have "Highlight group_id" instead of being exclusive (group_id or labels).

Reply:
Renamed variables!